app/javascript/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Greeting from './components/Greeting';
+import store from './Redux/configureStore';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <Router>
+        <Greeting />
```

Comment:
- [ Required ] I think it would be better if you set up a route in your `App` component that leads to the `Greeting` component. Currently, the `Greeting` component is not set up as a route in your `App` component. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-06-09 23-02-43](https://github.com/kennedyowusu/hello_rails-react/assets/16392046/6eefdd86-85fa-48dd-b15a-019ec1e23aa0)

Reply:
Can you come again on this? It is so not clear @Felix45

Reply:
I have resolved this issue as seen in the image below. Thank you

![3](https://github.com/kennedyowusu/hello_rails-react/assets/32777408/3fef8a7f-0393-4ceb-a122-53f52c8f3e29)

Reply:
I joined a call with the student and explained the changes